test/functional/wallet_importmulti.py

```diff
@@ -552,7 +552,7 @@ def run_test(self):
                                "keys": [key.privkey]},
                               success=False,
                               error_code=-5,
-                              error_message="Descriptor is invalid")
+                              error_message="Descriptor is invalid, Missing checksum")
```

Comment:
nit: make descriptor errors start with lower case?

Reply:
Nah.

Reply:
Agree with @Sjors, is there a reason to not do so? I was going to suggest to drop "Descriptor is invalid, " prefix.

Reply:
I've just removed the prefixes.

Reply:
They're back

Reply:
> They're back

No??